Sources/Hedera/Transaction.swift

```diff
@@ -479,9 +480,6 @@ extension Transaction {
 
 extension Transaction {
     fileprivate func makeTransactionList() throws -> [Proto_Transaction] {
-        assert(self.isFrozen)
```

Comment:
given the lack of a change to imply it's meant to be used otherwise, re-add this?